azure-pipelines.yml

```diff
@@ -68,7 +68,7 @@ jobs:
       cp ../*.deb .
     displayName: "Compile sonic sairedis"
   - script: |
-      make check || true
+      make check
```

Comment:
i think you can add sudo here if root is required to run the test.

Reply:
done

Reply:
reverted back since don't work